src/storage/src/table/streaming_table/state_table.rs

```diff
@@ -700,9 +700,69 @@ impl<S: StateStore> StateTable<S> {
                 .into_stream(),
         )
     }
+
+    /// This function scans rows from the relational table with specific `pk_prefix`.
+    pub async fn iter_with_pk<'a>(
```

Comment:
The name is confusing. The `pk_prefix` in `iter_with_pk_prefix` is argument, not return value.